js/npm/sequelize/DataTypes.hx

```diff
@@ -28,14 +28,14 @@ class DataTypes {
 	}
 	/**
 	 * A 32 bit integer. 
-	 * Available properties: `length`, `unsigned:Bool`, `zerofill:Bool`
+	 * Available properties: `length:Int`, `unsigned:Bool`, `zerofill:Bool`
 	 */
 	static public inline function INTEGER(?options:{}):DataTypes {

```

Comment:
also, why do you `static public inline` everything when you could just `static public` ?
finally, I don't understand why there is not `require`/`js.npm.Require` in this package.